code-challenges.js

```diff
@@ -33,22 +52,31 @@ const temp3 = 212
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
-
+// Pseudo code: 
+// The best way to get the length of an Array is to use the .length property. We can also assign a variable to that .length. We also need to use the .concat method to combine the two arrays.
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+const padres1998WorldSeries = padres1984WorldSeries.concat(padres1998WorldSeries.length)
+
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: 
+// The .reverse method is used to reverse letters. I will ned to use console.log to view the output. The variable has a name of "currentCohort" and it needs the .reverse in order to perform the action:
+console.log(currentCohort.reverse)
 
 const currentCohort = "Hotel 2022"
 // Expected output: "2202 letoH"
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: Looking at the elements in the array, I see that it has has an index of 8. It starts from 0, (13) and ends at 8 (10). The givenValue1 is set to 42, which lets me know that they're using [7]. 
+
+// console.log(myArray[7]) is what I was thinking but that would give an output of 42, instead of 7. 
```

Comment:
For this question, think about a method called .lastIndexOf(). It will search through an array of many duplicate values and return the last index of that value.